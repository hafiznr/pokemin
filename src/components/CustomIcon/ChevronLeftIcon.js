import React from 'react';

const ChevronLeftIcon = ({color, size}) => {
  return (
    <svg
      width={size || "24"}
      height={size || "24"}
      viewBox="0 0 24 24"
      fill="none"
      style={{display: 'inline-block'}}
    >
      <path
        d="M16.2426 6.34317L14.8284 4.92896L7.75739 12L14.8285 19.0711L16.2427 17.6569L10.5858 12L16.2426 6.34317Z"
        fill={color || 'currentColor'}
      />
    </svg>
  )
};

export default ChevronLeftIcon;