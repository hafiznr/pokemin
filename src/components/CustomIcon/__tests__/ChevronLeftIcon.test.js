import { render } from '@testing-library/react';
import ChevronLeftIcon from '../ChevronLeftIcon';

test('renders ChevronLeftIcon', () => {
  const { container } = render(<ChevronLeftIcon />);

  expect(container).toMatchSnapshot();
});
