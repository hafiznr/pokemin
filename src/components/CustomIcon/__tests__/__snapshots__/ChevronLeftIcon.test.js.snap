// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ChevronLeftIcon 1`] = `
<div>
  <svg
    fill="none"
    height="24"
    style="display: inline-block;"
    viewBox="0 0 24 24"
    width="24"
  >
    <path
      d="M16.2426 6.34317L14.8284 4.92896L7.75739 12L14.8285 19.0711L16.2427 17.6569L10.5858 12L16.2426 6.34317Z"
      fill="currentColor"
    />
  </svg>
</div>
`;
